scripts/Packages/Package_Gullwing__QFP_SOIC_SO/size_definitions/sop.yaml

```diff
@@ -2,6 +2,31 @@ FileHeader:
   library_Suffix: 'SO'
   device_type: 'SOP'
 
+
+SOP-8_4.14x5.33mm_P1.27mm:
+  size_source: 'https://ww2.minicircuits.com/case_style/XX211.pdf'
+  body_size_x:
+    nominal: 4.14
+    tolerance: 0.762
+  body_size_y:
+    nominal: 5.33
+    tolerance: 0.762
+  body_height:
+    max: 1.96
+  overall_size_x:
+    nominal: 6.35
+    tolerance: 0.762
```

Comment:
Dimensions D and E are for max/min of the overall size. So this should be minimum: 5.59 and maximum: 6.35.